### PR TITLE
OCPBUGS-18769: autoscaler fixes

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OCPBUGS-18769

Link to docs preview: https://68594--docspreview.netlify.app/openshift-enterprise/latest/networking/ingress-operator#nw-autoscaling-ingress-controller_configuring-ingress

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@melvinjoseph86 could you PTAL at this docs PR for QE ack? thank you

File: modules/nw-autoscaling-ingress-controller.adoc [hunk 1]
```diff
@@ -14,38 +14,6 @@ Automatically scale an Ingress Controller to dynamically meet routing performanc
 . You have the Custom Metrics Autoscaler Operator installed.
 
 .Procedure
-. Create a project in the `openshift-ingress-operator` namespace by running the following command:
```
Comment (melvinjoseph86) [inline modules/nw-autoscaling-ingress-controller.adoc:17]: This step should be re sentence as we should  be in the `openshift-ingress-operator` namespace before running the below steps. It should be 
`Switch to the openshift-ingress-operator namespace by running the following command:
$ oc project openshift-ingress-operator

**Review by melvinjoseph86 — COMMENTED** [review]
Also as the requestor asked we can modify the  Step 7, "Create a new ScaledObject YAML file, ingress-autoscaler.yaml, that targets the default Ingress Controller deployment".
`serverAddress: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091`
in the ingress-autoscaler.yaml file.

File: modules/nw-autoscaling-ingress-controller.adoc (hunk 1 above)
Comment (jldohmann) [reply modules/nw-autoscaling-ingress-controller.adoc:17]: gotcha, thank you. I will add this to the prerequisites section actually

**Comment by melvinjoseph86** [comment]
> @melvinjoseph86 could you PTAL at this docs PR for QE ack? thank you

Done

**Pull request merged**